rust/crd/src/history.rs

```diff
@@ -281,10 +281,10 @@ impl HistoryConfig {
             resources: ResourcesFragment {
                 cpu: CpuLimitsFragment {
                     min: Some(Quantity("200m".to_owned())),
-                    max: Some(Quantity("4".to_owned())),
+                    max: Some(Quantity("1".to_owned())),
```

Comment:
Marker to change to ration of 4 (and change in gsheet)

Reply:
done